packages/expo-symbols/src/SymbolModule.types.ts

```diff
@@ -53,11 +66,14 @@ export interface NativeSymbolViewProps extends ViewProps {
   animated: boolean;
   colors?: string | string[];
   tint?: string;
-  resizeMode?: SymbolContentMode;
+  resizeMode?: ContentMode;
   animationSpec?: AnimationSpec;
 }
 
-type SymbolWeight =
+/**
+ * @hidden Described in the {@link SymbolViewProps['weight']}
```

Comment:
```suggestion
 * Type description in here.
```

We should not hide the types like that. There will be still has link generated to those types It's recommended to leave those most of the API visible in general, unless there is something that we really have a reason to hide.

This suggestion also applies for the following types.

Reply:
Thanks for the context on that, I did not put that much attention to exposed API changes lately, but will check out how it's organized in `expo-image`. 

Also, please make sure that there are no warnings about missing link in the docs console when visiting the new docs page locally, refs:
* https://github.com/expo/expo/pull/27864